src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -298,6 +298,7 @@ private static boolean printStackTraceOnUserExceptions() {
     /**
      * Returns the command line program specified, or prints the usage and exits with exit code 1 *
      */
+    @SuppressWarnings("deprecation")
```

Comment:
I think most of the deprecation sites can be fixed by replace `class.newInstance()` with `class.getDeclaredConstructor().newInstance()` which seems to exist in 8.  It means adding additional exceptions to the list of exceptions caught or thrown but I think that's preferable to keeping the deprecated method and suppressing it.

Reply:
In intellij you can use find -> search structurally to find all these sites easily.

<img width="908" alt="Screen Shot 2019-09-13 at 3 25 40 PM" src="https://user-images.githubusercontent.com/4700332/64889359-2a251a80-d63b-11e9-8d14-a3bce15f8c0b.png">